cp-quickstart/stop-docker-cloud.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Source library 
+. ../utils/helper.sh
+
+if [ -z "$1" ]; then
+  echo "ERROR: Must supply argument that is the client configuration file created from './start.sh'. (Is it in stack-configs/ folder?) "
+  exit 1
+else
+  DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+  CONFIG_FILE=${DIR}/$1
+  ../ccloud/ccloud-stack/ccloud_stack_destroy.sh $CONFIG_FILE
+fi
+
+docker-compose down -v
```

Comment:
Maybe the docker-compose down should occur prior to the stack destroy.  Docker containers here are acting like clients, which probably should be destroyed before services.  They probably error our if the service is taken out from under them, which might cause confusion.  

I'm not sure if docker-compose down should run even if the user erroneously calls the function without the config file parameter, but if they are calling stop, at least clients would stop pumping data to the services, reducing potential unexpected data flow up to cloud.  Calling docker-compose down twice once the user realizes they didn't provide the config the first time is idempotent.

Reply:
@rspurgeon I went back and forth on this, but your last point is very compelling, and I switched this around (docker-compose down is first now).